account_payment_order/wizard/payment_order_create.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 brain-tec AG (http://www.braintec-group.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+
+from openerp import models, fields, api, _
+
+
+class PaymentOrderCreate(models.TransientModel):
+    """
+    Create a payment object with lines corresponding to the account move line
+    to pay according to the date and the mode provided by the user.
+    Hypothesis:
+    - Small number of non-reconciled move line, payment mode and
+    bank account type,
+    - Big number of partner and bank account.
+
+    If a type is given, unsuitable account Entry lines are ignored.
+    """
+
+    _name = 'payment.order.create'
+    _description = 'payment.order.create'
+
+    duedate = fields.Date('Due Date', required=True,
```

Comment:
As this is a TransientModel, there's no problem in renaming some columns, so I suggest to rename this to `due_date` for being more readable